### ISSUE TITLE
Migrate doesn't work with new dyno types

### ISSUE THREAD

**Issue by lukerandall** [daemonsy/hubot-heroku#17]
Running migrate on an app using the new dyno types fails:

```
luke: Telling Heroku to migrate app
luke: Shucks. An error occurred. 422 - No such size as 1X. Available dyno sizes are Free, Hobby, Standard-1X, Standard-2X and Performance.```

**Comment by daemonsy** [comment]
Hey @lukerandall previously I was hardcoding the dyno size to `1X` when running the migrate. It should be fixed with #18 

Thanks for reporting this!

**Comment by lukerandall** [comment]
Thanks for the fix!
